plugin.go

```diff
@@ -8,7 +8,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const pluginName = "github.com/monebag/monorepo-diff"
+const pluginName = "monorepo-diff-buildkite-plugin"
```

Comment:
I am happy to revert this change along with the `strings.Contains` below, but this seemed to be more flexible to me. Let me know what you think.

Reply:
I prefer to leave it as it is. Once you revert it back I will merge the PR